code/ModelAdmin.php

```diff
@@ -620,7 +620,7 @@ public function import($data, $form, $request)
 
         // File wasn't properly uploaded, show a reminder to the user
         if (empty($_FILES['_CsvFile']['tmp_name']) ||
-            file_get_contents($_FILES['_CsvFile']['tmp_name']) == ''
+            file_get_contents($_FILES['_CsvFile']['tmp_name'] ?? '') == ''
```

Comment:
An error is emitted whether this is an empty string or null.
> Uncaught ValueError: Path cannot be empty

Is a warning emitted _as well_ as the error if this is an empty string?

Reply:
both, php 8.1
```
php -a
php > file_get_contents(null);
PHP Deprecated:  file_get_contents(): Passing null to parameter #1 ($filename) of type string is deprecated in php shell code on line 1

Deprecated: file_get_contents(): Passing null to parameter #1 ($filename) of type string is deprecated in php shell code on line 1
PHP Warning:  Uncaught ValueError: Path cannot be empty in php shell code:1
Stack trace:
#0 php shell code(1): file_get_contents()
#1 {main}
  thrown in php shell code on line 1

Warning: Uncaught ValueError: Path cannot be empty in php shell code:1
Stack trace:
#0 php shell code(1): file_get_contents()
#1 {main}
  thrown in php shell code on line 1

```